test/e2e/config/default.json

```diff
@@ -3,6 +3,7 @@
 	"explicitWaitMS": 20000,
 	"emailWaitMS": 60000,
 	"mochaTimeoutMS": 160000,
+	"jestTimeoutMS": 160000,
```

Comment:
FYI this timeout is longer than the timeout currently used internally by Playwright (30s default). I am not sure what impact this may have; will check locally.